srv/modules/runners/changed.py

```diff
@@ -62,28 +104,42 @@ def rgw_configurations(self):
         try:
             roles = list(local.cmd("I@roles:master", 'pillar.get',
                                    ['rgw_configurations'],
-                                   tgt_type="compound").values())[0]
+                                   expr_form="compound").values())[0]
```

Comment:
Wait, what? I thought `expr_form` was the old name and `tgt_type was the new name?

Reply:
right, somehow my salt complained though.. changing it back to tgt_type